pkg/controllers/manifestwork.go

```diff
@@ -245,12 +245,12 @@ func (r *HypershiftDeploymentReconciler) createOrUpdateMainfestwork(ctx context.
 
 	passedSecurity, statusUpdateErr := r.validateSecurityConstraints(ctx, hyd)
 	if !passedSecurity {
-		return ctrl.Result{}, statusUpdateErr
+		return ctrl.Result{RequeueAfter: time.Minute * 1}, statusUpdateErr
```

Comment:
Just being picky. Do you think 1 minute is enough for someone to fix the clusterset binding issue?

Reply:
> Just being picky. Do you think 1 minute is enough for someone to fix the clusterset binding issue?

I think a minute is a reasonable time because if it's any longer the user might have to wait a long time before the next reconcile in the worst case scenario.